chains/orchestrator-paras/runtime/dancebox/src/lib.rs

```diff
@@ -286,6 +286,7 @@ pub const DAYS: BlockNumber = HOURS * 24;
 
 // Unit = the base number of indivisible units for balances
 pub const UNIT: Balance = 1_000_000_000_000;
+pub const CENTS: Balance = UNIT / 30_000;
```

Comment:
What is this unit? Divided by `30_000`? Can we stay metric-ish in our units? :p

Reply:
haha it comes from rococo, it's ok we can leave it like this or if we modify it we should do it as a separqate pr